src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutesSpec.scala

```diff
@@ -166,10 +163,10 @@ class ProxyRoutesSpec extends FlatSpec with BeforeAndAfterAll with BeforeAndAfte
     //  - `upgradeResponse` is a Future[WebSocketUpgradeResponse] that completes or fails when the connection succeeds or fails.
     //  - `result` is a Future[String] with the stream completion from the incoming sink.
     val (upgradeResponse, result) =
-      outgoing
-        .viaMat(webSocketFlow)(Keep.right)
-        .toMat(incoming)(Keep.both)
-        .run()
+    outgoing
```

Comment:
I think the indentation before was probably correct?
```
val (foo, bar) = 
  baz
```

Reply:
IntelliJ Auto Indent 👎